docs/data/data-grid/recipes-styling/HeaderCellFocusNoOutline.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { DataGridPro, gridClasses } from '@mui/x-data-grid-pro';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+const VISIBLE_FIELDS = ['name', 'rating', 'country', 'dateCreated', 'isAdmin'];
+
+export default function HeaderCellFocusNoOutline() {
```

Comment:
```suggestion
export default function CellFocusNoOutline() {
```

It impacts all cells, not only those in the header 🙂